php/providers/pecl.rb

```diff
@@ -102,20 +111,12 @@
 
 end
 
-def get_extension_dir
-  @extension_dir ||= begin
-    p = shell_out("#{new_resource.prefix}/bin/php-config --extension-dir")
-    p.stdout.strip
-  end
+def get_extension_dir(prefix)
```

Comment:
@fh Refactored this out for testing. (private methods in an LWRP are not mockable, because the LWRP is created at runtime, thus nothing can be mocked and stubbed beforehand.